aten/src/ATen/native/cuda/ForeachFunctors.cuh

```diff
@@ -547,5 +548,19 @@ struct TernaryOpScalarFunctor {
   }
 };
 
+template <typename T>
+struct power_functor {
+  C10_DEVICE T operator()(const T& a, const T& b) const {
+    return pow(a, b);
```

Comment:
what's the name resolution here?

Reply:
I meant cuda's pow. updated it to `at::native::pow_`